.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: ruby
 rvm:
-- 2.3
+  - 2.3
+services:
+  - docker
 branches:
   only:
   - develop
-script: rake travis
+before_install:
+  - docker pull debezium/awestruct
+  - docker run --rm --user $UID:$GID -e LC_ALL=C.UTF-8 -e LANG=C.UTF-8 -v $(pwd):/site debezium/awestruct rake setup && rake travis
```

Comment:
@Naros Not sure about this one but as it is written I expect it will execute `rake setup` in context of Docker container and then `rake travis` as a process outside of container